.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and test
+
+on: push
+
+env:
+  GITHUB_CI: true
+
+jobs:
+  run-tests:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
```

Comment:
Maybe try to migrate into 1.20 too?

Reply:
Definitely. In separate PR. This is my next change